### PR TITLE
Release 0.13.1

### PR DESCRIPTION
**Pull request by str4d**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,14 +7,15 @@ and this project adheres to Rust's notion of
...
 ### Added
 - `bellman::groth16::batch::Verifier` now has a `verify_multicore` method (when
   the `multicore` feature is enabled) which will internally use the global rayon
   thread pool to parallelize the verification of a batch of proofs.
 
 ### Changed
-- The `multicore` feature now requires the `getrandom` feature of the
-  `rand_core` crate.
+- The `multicore` feature now enables the `getrandom` feature of the `rand_core`
+  crate.
```
Comment (str4d) [inline CHANGELOG.md:18]: I'm still on the fence about this, as it is a breaking change to anyone trying to use `bellman` on a platform that [`getrandom` doesn't support](https://docs.rs/getrandom/0.2.7/getrandom/#supported-targets). But it seems like there will be very few platforms that don't have a system RNG but _do_ support allocators, threads, and the standard library.

Comment (daira) [reply CHANGELOG.md:18]: The [register_custom_getrandom!](https://docs.rs/getrandom/0.2.7/getrandom/macro.register_custom_getrandom.html) macro can be used to register an RNG implementation, although I agree that would be a breaking change on such platforms.

**Review by daira — APPROVED** [review]
utACK

**Review by ebfull — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (str4d) [reply CHANGELOG.md:18]: Still, in this case the benefit of a point release outweights this I think; if someone complains then we can deal with it then.

**Pull request merged**